adoc/chapters/programming_interface.adoc

```diff
@@ -1197,263 +1197,315 @@ Returns an empty list if this platform does not contain any devices.
 [code]#memory_order::acquire#, [code]#memory_order::release#, and
 [code]#memory_order::acq_rel#.
 
-a@
-[source]
+'''
+
+.[apidef]#info::context::atomic_memory_scope_capabilities#
+[source,role=synopsis,id=api:info-context-atomic-memory-scope-capabilities]
 ----
-info::context::atomic_memory_scope_capabilities
+namespace sycl::info::context {
+struct atomic_memory_scope_capabilities {
+  using return_type = std::vector<memory_scope>;
+};
+} // namespace sycl::info::context
 ----
 
-    @ [.code]#std::vector<memory_scope>#
-   a@ Returns the set of memory scopes supported by atomic operations on all
-      devices in the context.  When a context returns a "wider" memory scope
-      in this set, it must also return all "narrower" memory scopes.  (See
-      <<sec:memory-scope>> for a definition of "wider" and "narrower" scopes.)
-      At a minimum, each context must support [code]#memory_scope::work_item#,
-      [code]#memory_scope::sub_group#, and [code]#memory_scope::work_group#.
+_Remarks:_ Template parameter to [api]#context::get_info#.
 
-a@
-[source]
+_Returns:_ The set of memory scopes supported by atomic operations on all
+devices in the context.
+When a context returns a "wider" memory scope in this set, it must also return
+all "narrower" memory scopes.
+(See <<sec:memory-scope>> for a definition of "wider" and "narrower" scopes.)
+At a minimum, each context must support [code]#memory_scope::work_item#,
+[code]#memory_scope::sub_group#, and [code]#memory_scope::work_group#.
+
+'''
+
+.[apidef]#info::context::atomic_fence_scope_capabilities#
+[source,role=synopsis,id=api:info-context-atomic-fence-scope-capabilities]
 ----
-info::context::atomic_fence_scope_capabilities
+namespace sycl::info::context {
+struct atomic_fence_scope_capabilities {
+  using return_type = std::vector<memory_scope>;
+};
+} // namespace sycl::info::context
 ----
 
-    @ [.code]#std::vector<memory_scope>#
-   a@ Returns the set of memory orderings supported by [code]#atomic_fence# on
-      all devices in the context.  When a context returns a "wider" memory
-      scope in this set, it must also return all "narrower" memory scopes.
-      (See <<sec:memory-scope>> for a definition of "wider" and "narrower"
-      scopes.) At a minimum, each context must support
-      [code]#memory_scope::work_item#, [code]#memory_scope::sub_group#, and
-      [code]#memory_scope::work_group#.
-
-|====
+_Remarks:_ Template parameter to [api]#context::get_info#.
 
+_Returns:_ The set of memory orderings supported by [code]#atomic_fence# on all
+devices in the context.
+When a context returns a "wider" memory scope in this set, it must also return
+all "narrower" memory scopes.
+(See <<sec:memory-scope>> for a definition of "wider" and "narrower" scopes.)
+At a minimum, each context must support [code]#memory_scope::work_item#,
+[code]#memory_scope::sub_group#, and [code]#memory_scope::work_group#.
 
+'''
 
 [[sec:context-properties]]
-==== Context properties
+==== Properties
```

Comment:
Why `=== Context class` and just `Properties`?

Reply:
All of the reformatted PRs are like this.  The top-level section is named "XXX class", and the sub-sections are named like "Constructors", "Member functions", etc.  I decided not to name them "XXX constructors", etc. because that seemed redundant.

Reply:
Oh I see `4.6.3. Context class` and `property` is a level lower than that.
No hard thought about it, just need to be consistent 
I'm in slight favor of keeping `context`, make ctrl+f easier on the Table of Contents. But you already did the work of removing it everywhere, so it is not worth the effort of putting it back IMO.